src/components/EventDateFilters.vue

```diff
@@ -1,62 +1,92 @@
 <template>
-  <div>
+  <div class='event-date-filters'>
     <h5 v-if="showTitle" class="filter-events-title">Date of event</h5>
     <div class="filter-events-date-picker">
-      <input ref="startDate" type="text" :value="startDate" />
-      <strong>to</strong>
-      <input ref="endDate" type="text" :value="endDate" />
+      <div class='date-picker-header'>
+        <input
+          autofocus
+          v-on:click="currentCalendar = 'startDate'"
+          ref="startDate"
+          type="text"
+          :value="startDate"
+        />
+        <strong>to</strong>
+        <input
+          v-on:click="currentCalendar = 'endDate'"
+          ref="endDate"
+          type="text"
+          :value="endDate"
+        />
+      </div>
+      <div id='calendar-container'>
+        <div v-show="currentCalendar === 'startDate'" id='start-calendar' />
+        <div v-show="currentCalendar === 'endDate'" id='end-calendar' />
+      </div>
     </div>
   </div>
 </template>
 
 <script>
 import moment from 'moment';
+import Pikaday from 'pikaday';
 
 // sigh
 window.moment = moment;
 
-import Pikaday from 'pikaday';
```

Comment:
I think the reason I imported it that way was so that Pikaday would know that moment was available....I read somewhere that that was needed. But maybe it waits to check for moment until actually creating a Pikaday object? Anyway, if it works like this, cool. Might even be able to drop the `window.moment`, that would be nice...